### DISCUSSION TITLE
How do I change the URL created by the double bracket links?

### DISCUSSION THREAD

**Discussion by bobbieysabel** [binyamin/eleventy-garden#45] [Q&A] [answered]
Spun up my own digital garden and attempting to make it work with Netlify CMS. [repo](https://github.com/bobbieysabel/ciphermuses). [demo](https://blissful-pike-33323e.netlify.app).

If I use the CMS to create a new note with the title "Test Note", the resulting note has a URL ending in `notes/test-note/`.` See [here](https://blissful-pike-33323e.netlify.app/notes/test-note/).

However, writing `[[test note]]` in another note creates a link to a URL ending in `notes/test%20note`. See [here](https://blissful-pike-33323e.netlify.app/notes/). How do I adjust the URL created by the double brackets so that spaces become `-` instead of `%20`?

Thanks in advance.

**Comment by binyamin** [comment, marked as answer]
You would need to edit the RegExp in both `notes/notes.11tydata.js` and `.eleventy.js`. If Netlify CMS uses dashes instead of spaces, I should probably do the same. I'll push a fix tonight.

**Reply by binyamin** [reply]
I was wrong about the regex piece. I'll still take a look.

**Reply by binyamin** [reply]
Take a look at [.eleventy.js line 20](https://github.com/b3u/jubilant-pancake/blob/master/.eleventy.js#L20) and [notes/notes.11tydata.js line 33](https://github.com/b3u/jubilant-pancake/blob/master/notes/notes.11tydata.js#L33) in [b3u/jubilant-pancake](https://github.com/b3u/jubilant-pancake). I didn't push to the actual repo, because many people create their files in a writing app like obsidian.md which encourages spaces.

**Reply by bobbieysabel** [reply]
Works for me. Thanks for the help!
